package/PKGBUILD

```
@@ -2,38 +2,31 @@
 #
 # SPDX-License-Identifier: AGPL-3.0-only
 
-targets=(
-  "centos"
-  "ubuntu"
-)
 pkgname="carbonio-tasks-db"
 pkgver="0.0.1"
 pkgrel="1"
 pkgdesc="Carbonio Tasks database sidecar"
-pkgdesclong=(
-  "Carbonio Tasks database sidecar"
-)
 maintainer="Zextras <packages@zextras.com>"
 url="https://zextras.com"
 section="mail"
 priority="optional"
-arch="amd64"
+arch=('x86_64')
 license=("spdx:AGPL-3.0-only")
```

Since we are touching the PKGBUILD, please remove also the `spdx:` keyword
```suggestion
license=("AGPL-3.0-only")
```

---

Done, also added copyright metadatas